$OpenBSD: patch-Portroach_SiteHandler_Launchpad_pm,v 1.1 2017/03/25 14:18:49 jasper Exp $

Unbreak with perl 5.24:
Experimental keys on scalar is now forbidden at Portroach/SiteHandler/Launchpad.pm line 119.

--- Portroach/SiteHandler/Launchpad.pm.orig	Sat Mar 25 15:12:50 2017
+++ Portroach/SiteHandler/Launchpad.pm	Sat Mar 25 15:13:06 2017
@@ -116,7 +116,7 @@ sub GetFiles
 
 	    # 'entries' is a singleton array, where the first element
 	    # contains hashes with the actual entries
-	    foreach my $e (keys($entries{entries})) {
+	    foreach my $e (keys(%{$entries{entries}})) {
 		my $files_collection_link = $entries{entries}[$e]->{files_collection_link};
 
 		# Now that we have the files_collection_link, retrieve that so
@@ -128,7 +128,7 @@ sub GetFiles
 		if ($fcl_resp->is_success) {
 		    my %entries_fcl = %{decode_json($fcl_resp->decoded_content)};
 
-		    foreach my $ef (keys($entries_fcl{entries})) {
+		    foreach my $ef (keys(%{$entries_fcl{entries}})) {
 			my $self_link = $entries_fcl{entries}[$ef]->{self_link};
 			push @$files, $self_link;
 		    }
